torch/testing/_internal/common_methods_invocations.py

```diff
@@ -341,6 +348,60 @@ def __init__(self,
         #   outside a function's domain.
         self._domain_eps = 1e-5
 
+    def sample_sparse_inputs(self, device, dtype, requires_grad=False):
+        """Returns an iterable of SampleInputs."""
+        if not self.supports_sparse:
+            raise RuntimeError("Sparse layout not supported for ", self.name)
+
+        low, high = self.domain
+        low = -10 if low is None else low + self._domain_eps
+        high = 10 if high is None else high - self._domain_eps
+
+        numel = 10
+        t1 = torch.sparse_coo_tensor(
+            indices=torch.arange(numel).unsqueeze(0),
+            values=torch.linspace(low, high, steps=numel),
```

Comment:
Let's simplify this function and add a TODO to extend make_tensor to handle sparse tensors.

Until make_tensor is improved, however, just create a few sparse tensors manually here. make_tensor() for the values tensors and create the indices manually. The sparse tensors should be empty (no values), be 1D, and be 2D. Just a few specified values in each of the 1D and 2D tensors should be fine.

Reply:
@mruberry Have put in simple code to generate the tensors as suggested. Put a TODO for a more generic make_tensor kind of implementation